jax/_src/api.py

```diff
@@ -3267,7 +3267,7 @@ def try_to_block(x):
   return jax.tree_util.tree_map(try_to_block, x)
 
 def pure_callback(callback: Callable[..., Any], result_shape_dtypes: Any,
-                  *args: Any, **kwargs: Any):
+                  *args: Any, vectorized: bool = False, **kwargs: Any):
```

Comment:
I know this is already baked into the API, but should we be concerned that it's impossible to callback to a function that takes `callback`, `result_shape_dtypes`, or `vectorized` as a keyword argument?

Reply:
I agree it's not the most elegant solution, though users can always partial those arguments.